subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -9,6 +9,7 @@
 
 #include <zephyr.h>
 #include <stdio.h>
+#include <lte_lc.h>
```

Comment:
Not needed if you won't be doing a `lte_lc_offline`/`lte_lc_connect`.

What I found is that you get a 
```
[00:00:55.537,261] <err> asset_tracker: Socket error: POLLHUP
[00:00:55.543,975] <err> asset_tracker: Connection was closed by the cloud.
[00:00:55.551,818] <err> asset_tracker: LTE link disconnect
``` 
If you do it so I guess some more state handling/shifting is needed.

Reply:
After consulting those who are responsible for `asset_tracker` it was decided that a `lte_lc_power_off`would be the best solution